test/test_plugin_systemd.rb

```diff
@@ -17,21 +17,19 @@ def setup
 
     super
 
-    ::Dir::Tmpname.create("puma_socket") do |sockaddr|
-      @sockaddr = sockaddr
-      @socket = Socket.new(:UNIX, :DGRAM, 0)
-      socket_ai = Addrinfo.unix(sockaddr)
-      @socket.bind(socket_ai)
-      @env = {"NOTIFY_SOCKET" => sockaddr }
-    end
+    @sockaddr = tmp_path '.systemd'
+    @socket = Socket.new(:UNIX, :DGRAM, 0)
+    socket_ai = Addrinfo.unix(@sockaddr)
+    @socket.bind Addrinfo.unix(@sockaddr)
+    @env = {"NOTIFY_SOCKET" => @sockaddr }
```

Comment:
```suggestion
    @env = { "NOTIFY_SOCKET" => @sockaddr }
```

Reply:
As you suggested, and rebased.

Interesting, much of Ruby's docs have no spaces with hashes, but much of the Rails docs use spaces...